### ISSUE TITLE
Proposal for changes to the Infragram UI 

### ISSUE THREAD

**Issue by shubhangi013** [publiclab/infragram#210]
This is the issue that would contain all the changes I propose for Infragram  w.r.t : https://publiclab.org/wiki/gsoc-ideas#Outreachy:+Infragram.org+full-screen+UI+design.
I have experience with both designing and implementing wireframes. Since I’ve been working with Figma for over a year now, it is my go-to tool and via this issue, I’ll be updating all the changes I feel should be implemented in the UI of Infragram. 

Via #197 and #208 and linked Pull requests I’ve already suggested that we revamp the QnA and the footer. 
I’ll soon be working on the wireframe and further enhancing these changes.

**Comment by jywarren** [comment]
Hi, I appreciate the brainstorming! Would a full-screen layout, maybe like https://photopea.com/, help a lot? What do you think?

**Comment by shubhangi013** [comment]
> Hi, I appreciate the brainstorming! Would a full-screen layout, maybe like https://photopea.com/, help a lot? What do you think?

Yeah, it certainly looks better.  All the utilities can be shifted to a sidebar or a topbar.

**Comment by shubhangi013** [comment]
@TildaDares @jywarren 
Could you review this design? I will elaborate further on every window ASAP
